blueocean-pipeline-editor/src/main/js/services/PipelineValidator.js

```diff
@@ -20,6 +20,10 @@ export function isValidEnvironmentKey(key: string): boolean {
     return false;
 }
 
+function _isArray(o) {
+    return o instanceof Array || typeof o === 'array';
```

Comment:
If this data ever becomes managed by MobX just be aware that `ObservableArray` will fail these tests. May be fine to ignore for now.